packages/terra-functional-testing/src/terra-dev-site/terra-functional-testing/wdio-testing/About.tool.mdx

```diff
@@ -59,6 +57,24 @@ Tests can be executed in any of the following supported form factors:
 | huge     | 1300  | 768    |
 | enormous | 1500  | 768    |
 
+## Environment Variables
+
+The following environment variables are configurable for the test run. Variables marked as reserved will be overwritten by the test runner. To configure reserved variables use the test runner configuration options. If the test runner is not being used the variables are configurable by the user.
+
+| Variable Name                 | Type    | Reserved | Default             | Description                                                                                                                               |
+|-------------------------------|---------|----------|---------------------|-------------------------------------------------------------------------------------------------------------------------------------------|
+| BROWSERS                      | string  | true     | chrome              | A list of browsers for the test run. Browsers are delimited by a comma. Example: chrome,firefox,ie                                        |
+| FORM_FACTOR                   | string  | true     |                     | A form factor assigned to the test run.                                                                                                   |
+| LOCALE                        | string  | true     | en                  | A locale assigned to the test run.                                                                                                        |
+| SELENIUM_GRID_URL             | string  | true     |                     | The remote selenium grid address.                                                                                                         |
+| SITE                          | string  |          |                     | A file path to a static directory of assets. When defined, an express server will launch to serve the assets and disable running webpack. |
+| THEME                         | string  | true     | terra-default-theme | A theme name assigned to the test run.                                                                                                    |
+| WDIO_DISABLE_SELENIUM_SERVICE | boolean |          |                     | A flag to disable the selenium docker service.                                                                                            |
```

Comment:
Should we call out the default for WDIO_DISABLE_SELENIUM_SERVICE is `false`?

Reply:
I think that makes sense. I overthink some of these a little too much, but for all intents and purposes this is false (even though it's actually undefined/not set). I'll update this to false

Reply:
Update to default to false:

https://github.com/cerner/terra-toolkit/pull/544/commits/eb863d146f8d4cdc9ac5c7e661bd2830bf548a8f